spek-ide-plugin/intellij-common/src/main/kotlin/org/spekframework/intellij/domain/descriptors.kt

```diff
@@ -0,0 +1,226 @@
+package org.spekframework.intellij.domain
```

Comment:
this is a lot of code that seems easy to make a mistake in and hard to change in future, do you think there are reasonable ways to unit-test it?

Maybe something similar to [Android Lint tests](https://github.com/googlesamples/android-custom-lint-rules/blob/439605d4b10afa87cb58ba500d4c4b9a219770bf/android-studio-3/checks/src/test/java/com/example/lint/checks/SampleCodeDetectorTest.java) when you pipe source to it and run UAST/PSI analysis as a test

Reply:
Till today I still don't know how to test IJ plugins 😂 . But yeah, I'll add in more tests.

Reply:
No one does!